bibtexparser/bparser.py

```diff
@@ -75,13 +76,18 @@ def __init__(self, data=None,
                  homogenize_fields=False,
                  interpolate_strings=True,
                  common_strings=True,
-                 add_missing_from_crossref=False):
+                 add_missing_from_crossref=False,
+                 expect_multiple_parse=False):
```

Comment:
I suggest to *not* make this a constructor argument (instead users should modify the property).

Reasons: Keeping the constructor less verbose by not including options which will hardly ever be used.

Reply:
What do you mean by the property? property with using decorators like `@property` and etc. or an instance attribute like below code would be ok? and if I remove it from the constructor how can I pass it as a flag?

```
# Removing from the constructor arguments 
self.expect_multiple_parse = False
```

Usage:

```
parser = BibTexParser()
parser.expect_multiple_parse = False
```

Reply:
For now, your proposition is fine (that's how the remainder of this library, which dates many python versions back, is set up. in addition, if we ever need to change to `@property`, we will be able to do so without breaking backward compatibility.